2022/09/03 07:08:55 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/03 07:09:33 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "127.0.0.1"
2022/09/03 07:16:45 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/03 07:18:09 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "127.0.0.1"
2022/09/03 07:30:22 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/03 07:35:41 [error] 8#8: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "127.0.0.1"
2022/09/05 07:55:10 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/05 13:59:00 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "127.0.0.1"
2022/09/05 13:59:06 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "127.0.0.1"
2022/09/05 13:59:17 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/05 14:00:11 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/05 14:00:26 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/05 14:06:09 [error] 8#8: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/05 14:07:58 [error] 8#8: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:07:12 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:08:25 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:08:29 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:11:40 [error] 8#8: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:11:41 [error] 8#8: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/stylesheets?v=1657531602&theme=auto HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/"
2022/09/06 05:11:41 [error] 8#8: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/javascript?v=1657531602 HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/"
2022/09/06 05:28:10 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:28:13 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:28:16 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:28:19 [error] 8#8: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:39:48 [error] 8#8: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:40:02 [error] 8#8: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:40:11 [error] 8#8: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 05:54:33 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:02:27 [error] 8#8: *20 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:04:56 [error] 8#8: *20 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:06:00 [error] 8#8: *20 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:12:46 [error] 8#8: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:21:32 [error] 8#8: *26 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:35:40 [error] 8#8: *28 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 07:36:25 [error] 8#8: *32 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/06 07:36:27 [error] 8#8: *32 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/javascript?v=1657531602 HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1", referrer: "https://172.17.0.1/"
2022/09/06 07:36:27 [error] 8#8: *32 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/stylesheets?v=1657531602&theme=auto HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1", referrer: "https://172.17.0.1/"
2022/09/06 08:20:49 [error] 8#8: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 08:20:55 [error] 8#8: *38 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/06 08:22:34 [error] 8#8: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 08:24:40 [error] 8#8: *36 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 08:32:15 [error] 8#8: *42 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 08:32:27 [error] 8#8: *44 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/06 08:38:16 [error] 8#8: *46 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/06 08:40:02 [error] 8#8: *48 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 08:40:23 [error] 8#8: *46 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/06 09:50:26 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:06 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:08 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:09 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:09 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:09 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:09 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:10 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:10 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 09:52:10 [error] 8#8: *51 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:11:17 [error] 8#8: *64 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:12:32 [error] 8#8: *64 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:13:19 [error] 8#8: *64 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:20:49 [error] 8#8: *68 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:23:12 [error] 8#8: *68 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:24:58 [error] 8#8: *68 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/06 10:28:02 [error] 8#8: *72 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 10:37:44 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 10:37:49 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 10:44:31 [error] 7#7: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 10:48:26 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 10:48:27 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/stylesheets?v=1657531602&theme=auto HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/logger"
2022/09/07 10:48:27 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/javascript?v=1657531602 HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/logger"
2022/09/07 11:03:34 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 11:04:14 [error] 7#7: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), 172.17.01:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 12:54:14 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 12:54:17 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 12:54:19 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 12:54:25 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 12:54:54 [error] 7#7: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 12:54:56 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 12:54:57 [error] 7#7: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:08:08 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 13:08:14 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:08:15 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:10:35 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:11:13 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:15:58 [error] 7#7: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/07 13:26:15 [error] 7#7: *11 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:27:16 [error] 7#7: *11 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:28:24 [error] 7#7: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:28:24 [error] 7#7: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/stylesheets?v=1657531602&theme=auto HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/babelsort"
2022/09/07 13:28:24 [error] 7#7: *14 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/javascript?v=1657531602 HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/babelsort"
2022/09/07 13:28:32 [error] 7#7: *11 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:31:42 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:32:20 [error] 7#7: *19 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:47:42 [error] 7#7: *22 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:48:13 [error] 7#7: *22 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 13:56:55 [error] 7#7: *25 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/07 14:26:35 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:02:06 [error] 6#6: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:02:50 [error] 6#6: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:03:36 [error] 6#6: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:08:49 [error] 6#6: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:11:45 [error] 6#6: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:12:03 [error] 6#6: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:13:41 [error] 6#6: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:14:21 [error] 6#6: *13 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:15:11 [error] 6#6: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:15:23 [error] 6#6: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:15:58 [error] 6#6: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:17:18 [error] 6#6: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:17:43 [error] 6#6: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:22:47 [error] 6#6: *21 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 07:23:37 [error] 6#6: *21 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 07:31:12 [error] 6#6: *24 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:04 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:28 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:33 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:36 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:41 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:35:52 [error] 7#7: *7 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 07:36:12 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:40:22 [error] 7#7: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 07:57:23 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:05:07 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:05:26 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:08:15 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:10:25 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:10:29 [error] 8#8: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 08:11:24 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:11:26 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:11:49 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:11:51 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:12:43 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:12:45 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:12:51 [error] 8#8: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 08:12:57 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:13:42 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:13:44 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:14:24 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/"
2022/09/08 08:14:58 [error] 8#8: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/"
2022/09/08 08:16:34 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 08:16:47 [error] 8#8: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 08:17:43 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/08 08:17:47 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost", referrer: "https://localhost/"
2022/09/08 08:20:08 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /sort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:20:24 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /sort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:21:30 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:22:18 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:24:48 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:25:14 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /logger HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/08 08:26:21 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:43:35 [error] 8#8: *2 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/09 14:44:06 [error] 8#8: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:44:59 [error] 8#8: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:45:04 [error] 8#8: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:45:12 [error] 8#8: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:51:33 [error] 8#8: *11 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:57:29 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 14:58:00 [error] 7#7: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/09 15:10:29 [error] 7#7: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/09 15:10:32 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 15:33:59 [error] 7#7: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 15:34:12 [error] 7#7: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /babelsort HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/09 15:34:17 [error] 7#7: *10 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/11 07:41:33 [error] 7#7: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "localhost"
2022/09/11 07:41:39 [error] 7#7: *3 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:13:02 [error] 7#7: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:20:34 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:22:20 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:22:41 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:23:25 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET /_debugbar/assets/javascript?v=1657531602 HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:23:29 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:23:41 [error] 7#7: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:34:00 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:35:49 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:36:56 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:37:41 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:37:49 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 09:38:27 [error] 7#7: *15 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.5:9000", host: "172.17.0.1"
2022/09/11 14:03:45 [error] 6#6: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.7:9000", host: "172.17.0.1"
2022/09/12 08:44:09 [error] 6#6: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.7:9000", host: "172.17.0.1"
2022/09/12 08:44:11 [error] 6#6: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.7:9000", host: "172.17.0.1"
2022/09/12 08:44:16 [error] 6#6: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.7:9000", host: "localhost"
2022/09/12 08:44:19 [error] 6#6: *8 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.20.0.1:9003 (from REMOTE_ADDR HTTP header), host.docker.internal:9003 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.20.0.1, server: torg, request: "GET / HTTP/2.0", upstream: "fastcgi://172.20.0.7:9000", host: "localhost"
